Integrations/integration-Netskope.yml

```diff
@@ -0,0 +1,307 @@
+                    "Netskope.Events.url" : retArray.URL
+               }
+            }
+    }
+  type: javascript
+  commands:
+  - name: NetskopeEvents
+    arguments:
+    - name: query
+      description: filter query (e.g. user eq jtesarz@netskope.com)
+    - name: timeperiod
+      auto: PREDEFINED
+      predefined:
+      - "3600"
+      - "86400"
+      - "604800"
+      - "2592000"
+      - "5184000"
+      - "7776000"
+      description: TimePeriod
+    - name: starttime
+      description: start time in epoch
+    - name: endtime
+      description: end time in epoch
+    - name: type
+      required: true
+      auto: PREDEFINED
+      predefined:
+      - application
+      - page
+      - audit
+      description: Type of Event (e.g Application, Page, Audit)
+    - name: limit
+      description: Limit the number of events returned (useful for pagination in combination
+        with skip). Positive integer less than 5000.
+    - name: skip
+      description: Skip over some of the events (useful for pagination in combination
+        with limit)
+    outputs:
+    - contextPath: Netskope.Events.app
+      description: Name of Application
+    - contextPath: Netskope.Events.timestamp
+      description: Timestamp of event
+    - contextPath: Netskope.Events.activity
+      description: Activity of event
+    - contextPath: Netskope.Events.object
+      description: Document/object from event
+    - contextPath: Netskope.Events.hostname
+      description: Hostname of device
+    - contextPath: Netskope.Events.Category
+      description: Netskope application category (e.g. Cloud Storage, Webmail, etc)
+    - contextPath: Netskope.Events.device_classification
+      description: Device classification (e.g. managed vs unmanaged)
+    - contextPath: Netskope.Events.user
+      description: User
+    - contextPath: Netskope.Events.from_user
+      description: Login IDs for cloud apps
+    - contextPath: Netskope.Events.to_user
+      description: Destination user IDs
+    - contextPath: Netskope.Events.srcip
+      description: Source IP
+    - contextPath: Netskope.Events.access_method
+      description: Access method (e.g. client, reverse proxy, Secure Forwarder, etc)
+    - contextPath: Netskope.Events.url
+      description: URL
+    - contextPath: Netskope.Events.appcategory
+  - name: NetskopeAlerts
+    arguments:
+    - name: type
+      required: true
+      auto: PREDEFINED
+      predefined:
+      - anomaly
+      - Compromised Credential
+      - DLP
+      - Legal Hold
+      - malsite
+      - Malware
+      - policy
+      - quarantine
+      - Remediation
+      - watchlist
+    - name: timeperiod
+      auto: PREDEFINED
+      predefined:
+      - "3600"
+      - "86400"
+      - "604800"
+      - "2592000"
+      - "5184000"
+      - "7776000"
+      description: Time period (e.g. Last 60 minutes, Last 24 hours, etc.)
+    - name: starttime
+      description: Start time in epoch
```

Comment:
It would be better user experience if the user will pass the date in human readable format and the integration would convert the date to epoch time and send to netskope.